2024-01-31 22:39:45 [DEBUG] common_utils - <module> line 39: Config(env='local', common_config={'logger_path': {'LOGGER_DEF_FILE': 'tests/def/mfaUtilsLoggingHelper/logging_TimedRotate.json', 'LOGGER_MSG_FILE': 'tests/def/mfaUtilsLoggingHelper/config_MessageList.toml'}, 'input_file_path': {'UPDATE_EXCEL_PATH': 'tests/share/receive'}, 'output_file_path': {'SEND_REFFERENCE_MASTER_PATH': 'tests/share/receive'}}, package_config={'test': {'local_test': 'test'}}, log_message=<bound method LoggerPackage.log_message of <src.lib.common_utils.ibr_logger_package.LoggerPackage object at 0x7f2fad976250>>):
2024-01-31 22:39:45 [DEBUG] common_utils - test__UT_C0_normal line 74: 
        - テスト関数: test__UT_C0_normal
        - テスト定義:
                - テストカテゴリ: C0
                - テスト区分: 正常系/UT
                - テストシナリオ:
        :
2024-01-31 22:39:45 [DEBUG] common_utils - test_create_message_UT_C0_normal line 104: 
        - テスト関数: test_create_message_UT_C0_normal, test_send_email_via_smpt_UT_C0_error
        - テスト定義:
                - テストカテゴリ: C0
                - テスト区分: 正常系/UT
                - テストシナリオ: メッセージがMIMEフォマット生成される
        :
2024-01-31 22:39:45 [DEBUG] common_utils - test_send_email_via_smpt_UT_C0_error line 141: 
        - テスト関数: test_create_message_UT_C0_normal, test_send_email_via_smpt_UT_C0_error
        - テスト定義:
                - テストカテゴリ: C0
                - テスト区分: 正常系/UT
                - テストシナリオ: send_message()呼び出し挙動確認
                    ただしSMTPサーバ存在しない環境での確認故に
                    エラーが返ってくる前提での検証になっている
        :
2024-01-31 22:40:05 [DEBUG] common_utils - <module> line 39: Config(env='local', common_config={'logger_path': {'LOGGER_DEF_FILE': 'tests/def/mfaUtilsLoggingHelper/logging_TimedRotate.json', 'LOGGER_MSG_FILE': 'tests/def/mfaUtilsLoggingHelper/config_MessageList.toml'}, 'input_file_path': {'UPDATE_EXCEL_PATH': 'tests/share/receive'}, 'output_file_path': {'SEND_REFFERENCE_MASTER_PATH': 'tests/share/receive'}}, package_config={'test': {'local_test': 'test'}}, log_message=<bound method LoggerPackage.log_message of <src.lib.common_utils.ibr_logger_package.LoggerPackage object at 0x7f900c5f4690>>):
2024-01-31 22:40:05 [DEBUG] common_utils - test__UT_C0_normal line 74: 
        - テスト関数: test__UT_C0_normal
        - テスト定義:
                - テストカテゴリ: C0
                - テスト区分: 正常系/UT
                - テストシナリオ:
        :
2024-01-31 22:40:05 [DEBUG] common_utils - test_create_message_UT_C0_normal line 104: 
        - テスト関数: test_create_message_UT_C0_normal, test_send_email_via_smpt_UT_C0_error
        - テスト定義:
                - テストカテゴリ: C0
                - テスト区分: 正常系/UT
                - テストシナリオ: メッセージがMIMEフォマット生成される
        :
2024-01-31 22:40:05 [DEBUG] common_utils - test_send_email_via_smpt_UT_C0_error line 141: 
        - テスト関数: test_create_message_UT_C0_normal, test_send_email_via_smpt_UT_C0_error
        - テスト定義:
                - テストカテゴリ: C0
                - テスト区分: 正常系/UT
                - テストシナリオ: send_message()呼び出し挙動確認
                    ただしSMTPサーバ存在しない環境での確認故に
                    エラーが返ってくる前提での検証になっている
        :
